### PR TITLE
May 2016 Meetup

### PR DESCRIPTION
**Pull request by ryan-roemer**
List of things before final:

* [x] @hamoore 's currently waiting on "Waiting on info from Chris, decision from Madrona, sponsor"
* [x] Rename file to `2016-04-26-May-2016-koa-and-nwjs` (lower cased and "slug like"). We'll wait to do this until right before publishing as we need to bump the date too.

/cc @hamoore @scottnonnenberg

### CODE REVIEW

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "May 2016 Meetup - Case studies in Koa.js and NWjs"
+description: ""
+tags: [node.js, javascript, security]
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: Remove `security`, add `koa, nwjs`

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+{% include JB/setup %}
+
+The Seattle Node.js Meetup is pleased announce our
+[meetup](http://www.meetup.com/Seattle-Node-js/events/229425709/)
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: Leave as `[meetup](XXX)` until we have a real meetup event link.

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+on **(date needed)** with a look case studies using Koa.js and NWjs.
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: TODO: Fill in before final.

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+As always, you can find all things Seattle Node.js-related at the group website:
+[http://seanode.github.io/](http://seanode.github.io/). And you can reach out to
+us via email at [{{site.author.email}}](mailto:{{site.author.email}}) or Twitter
+at [@{{site.author.twitter}}](https://twitter.com/{{ site.author.twitter }}).
+
+See you soon!
+
+## Agenda
+
+**6:30 - 7:00pm**: Start with beverages and food, courtesy of NEED SPONSOR.
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: TODO: Fill in before final.

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+**7:00pm - 8:30pm**: Listen and learn about using Koa.js and generators as a happy path to async/await and bundling an application for native desktops with NWjs.
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: Shorten to:

```
Listen and learn about using Koa.js / generators and bundling desktop apps with NWjs.
```

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+<!-- more start -->
+
+## Talks
+
+### Koa.js: a happy path to async/await
+
+**[Chris Griffing](http://chrisgriffing.com/)** - Software Developer [Fresh Consulting](http://www.freshconsulting.com/)
+
+With the future of Express uncertain, a lot of people are looking for another framework to take its place. There are numerous options to choose from with the top 2 being Hapi.js and Koa.js. I'm not going to claim one is better than the other because the Hapi ecosystem is awesome and thriving. However, I do think Koa provides an easier upgrade path to async/await once it is supported in Node.
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: GLOBAL: Please line break paragraphs at 80 characters.

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+**[Rick Patci](NEED LINK)** - Assistant Instructor [Code Fellows](https://www.codefellows.org/)
+
+NEED DESCRIPTION 
+
+NEED BIO
+
+## Venue / Sponsor
+
+[Madrona](http://www.cdkglobaldigitalmarketing.com/) has graciously agreed to
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: Link is incorrect. Please switch to whole section to `NEED_SPONSOR` for now.

File: _posts/2016-04-26-May-2016-Koa.js-&-NWjs [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+**[Chris Griffing](http://chrisgriffing.com/)** - Software Developer [Fresh Consulting](http://www.freshconsulting.com/)
+
+With the future of Express uncertain, a lot of people are looking for another framework to take its place. There are numerous options to choose from with the top 2 being Hapi.js and Koa.js. I'm not going to claim one is better than the other because the Hapi ecosystem is awesome and thriving. However, I do think Koa provides an easier upgrade path to async/await once it is supported in Node.
+
+Chris Griffing is a full stack developer at Fresh Consulting in Bellevue. His hobbies include snowboarding, golfing, and going to meetups. Some say that his beanie is physically attached to his scalp.
+
+### Something about NWjs
+
+**[Rick Patci](NEED LINK)** - Assistant Instructor [Code Fellows](https://www.codefellows.org/)
```
Comment (ryan-roemer) [inline _posts/2016-04-26-May-2016-Koa.js-&-NWjs]: TODO: Fill in before final.

**Comment by ryan-roemer** [comment]
@hamoore -- Great first draft! First pass review done. Ping for re-review.

**Comment by hamoore** [comment]
@ryan-roemer - Ready for re-review.

**Comment by ryan-roemer** [comment]
@hamoore -- Nice edits! I've switched to a bullet list here with this commit https://github.com/seanode/seanode.github.io/pull/9/commits/4f536f317a02effc4321b569dbd5515b977f36fd

We are now in holding pattern until we get the rest of our information.

**Comment by hamoore** [comment]
Host/Sponsor and date updated.

**Comment by hamoore** [comment]
@ryan-roemer All info updated and ready for review!

**Comment by ryan-roemer** [comment]
Finished reviewing and we're good to go. Thanks @hamoore !

**Pull request merged**